ravenframework/Models/HybridModels/HybridModelBase.py

```diff
@@ -161,9 +161,32 @@ def submit(self,myInput,samplerType,jobHandler,**kwargs):
     ## Hybrid models need access to the job handler, so let's stuff it in our
     ## catch all kwargs where evaluateSample can pick it up, not great, but
     ## will suffice until we can better redesign this whole process.
-    prefix = kwargs['prefix']
-    kwargs['jobHandler'] = jobHandler
-    jobHandler.addClientJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, prefix, kwargs)
+
+    nRuns = 1
```

Comment:
These modifications are needed to make the HybridModel work with GeneticAlgorithms